### DISCUSSION TITLE
What happens if I hit Ctrl-C?

### DISCUSSION THREAD

**Discussion by Dani21** [Tyrrrz/DiscordChatExporter#618] [Q&A] [answered]
I'm downloading media as part of the export and didn't realize how large one of the channels was, so I want to re-run the CLI app under different settings mid-way through an export. What happens if I hit Ctrl-C to cancel? Will it save its progress and gracefully close out? This seems especially relevant since I noticed the size is 0 for files currently being worked on, so I don't want to lose the work completed so far.

**Comment by Yudi** [comment, marked as answer]
It should generate the file when you hit CTRL+C. Some stuff might break depending on the format, but you'll be able to open the file with a text editor.
You won't be able to reuse the exported file to resume the export, but you can use date ranges.
